xbmc/pvr/PVRContextMenus.cpp

```diff
@@ -218,7 +219,7 @@ namespace PVR
 
     bool FindSimilar::Execute(const CFileItemPtr& item) const
     {
-      return CServiceBroker::GetPVRManager().GUIActions()->FindSimilar(item);
+      return CServiceBroker::GetPVRManager().Get<PVR::GUI::EPG>().FindSimilar(item);
```

Comment:
A general question is this form of `Get<...>()` going to be use to get any component from the PVR Manager, GUI component or anything else?